src/data/faq.json

```
@@ -55,7 +55,7 @@
                             "The best place to share the unique report ID is where it was requested. You can also share the ID in the following ways:",
                             "<ul><li>Open an issue on GitHub (<a href='https://github.com/corona-warn-app/cwa-app-android/issues/new/choose' target='_blank' rel='noopener noreferrer'>Android</a> or <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/new/choose' target='_blank' rel='noopener noreferrer'>iOS</a>) and mention the ID of your error report</li><li> Share the ID with the technical support</li><li> Write a review on the Corona-Warn-App in the Apple App Store or Google Play Store and mention the ID of your error report</ul></li>",
                             "Regardless of which contact option you choose, the following tips will help to resolve your issue as quickly as possible:",
-                            "<ol><li>Describe the problem as accurately as possible</li><li>Start the error analysis in the CWA (<a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>see blog post</a>)<ul><li>Try to reproduce the incorrect behaviour or to generate the error message</li><li>If this is successful, send the error report to the CWA server. The data will be anonymized and encrypted</li><li>You can then find your unique ID on the bottom of the 'Error Reports' screen, scroll down and click on 'ID History'</li></ul></li><li>Stop the error analysis</li><li>Share the 'unique report ID' with the technical support</li></ol>",
+                            "<ol><li>Describe the problem as accurately as possible</li><li>Start the error analysis in the CWA (<a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>see blog post</a>)<ul><li>Try to reproduce the incorrect behaviour or to generate the error message</li><li>If this is successful, send the error report to the CWA server. The data will be anonymized and encrypted</li><li>You can then find your unique ID on the bottom of the 'Error Reports' screen, scroll down and click on 'ID History'</li></ul></li><li>Stop the error analysis</li><li>Share the 'unique report ID' with the technical support. On Android you can  long press the ID and copy & paste it like normal text.</li></ol>",

```

```suggestion
                            "<ol><li>Describe the problem as accurately as possible</li><li>Start the error analysis in the CWA (<a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>see blog post</a>)<ul><li>Try to reproduce the incorrect behaviour or to generate the error message</li><li>If this is successful, send the error report to the CWA server. The data will be anonymized and encrypted</li><li>You can then find your unique ID on the bottom of the 'Error Reports' screen, scroll down and click on 'ID History'</li></ul></li><li>Stop the error analysis</li><li>Share the 'unique report ID' with the technical support. On Android you can long press the ID and copy & paste it like normal text.</li></ol>",
```
remove extra space between "can" and "long"